### PR TITLE
Add component lists

### PR DESCRIPTION
**Pull request by sah-shah7**
-Added List Component Screen
image

### CODE REVIEW

File: Components/Lists/list.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+.stack-list-item{
+    position: relative;
+    display: block;
+    padding: 0.5rem 1rem;
+    color: #212529;
+    text-decoration: none;
+    background-color: #fff;
+    border: 1px solid rgba(0,0,0,.125);
```
Comment (JatuFaique) [inline Components/Lists/list.css:55]: Use consistent units for color or else create variables for color

**Review by JatuFaique — APPROVED** [review]
Looks good to merge

**Review by shivamSspirit — APPROVED** [review]
proper nice code

File: Components/Lists/list.css (hunk 1 above)
Comment (sah-shah7) [reply Components/Lists/list.css:55]: Thanks will factor this for the next pr

**Comment by sah-shah7** [comment]
PR approved. Merging.

**Pull request merged**